freezed_provider_value_notifier/test/app_state_test.dart

```diff
@@ -0,0 +1,159 @@
+// Copyright 2018 The Flutter Architecture Sample Authors. All rights reserved.
+// Use of this source code is governed by the MIT license that can be found
+// in the LICENSE file.
+
+import 'package:freezed_provider_value_notifier/models.dart';
+import 'package:freezed_provider_value_notifier/todo_list_model.dart';
+import 'package:test/test.dart';
+
+import 'mock_repository.dart';
+
+void main() {
+  group('TodoListModel', () {
```

Comment:
Feels like Model and Controller tests are being mixed here, would recommend splitting this into two groups: One that tests the Model and another that tests the functionality of the Controller.

Reply:
Ah, I didn't see that.
Done

Reply:
I dig it. Adding `protected` also makes me happy, since I did notice the `value` could have been changed by anyone at any time from an outside source.